tests/test_core.py

```diff
@@ -60,7 +60,7 @@ def test_coreRecordAsSolrDoc():
 }    
     """
     solr_dict = _try_to_add_solr_doc(core_doc_str)
-    assert "producedBy_samplingSite_location_latlon" in solr_dict
+    assert "producedBy_samplingSite_location_ll" in solr_dict
```

Comment:
This test was broken by the change to use the RPT fields.  Long-term we need to get some kind of CI setup going on GitHub.